### PR TITLE
Completed UI design & CSS!

### PR DESCRIPTION
**Pull request by kvtim**

### CODE REVIEW

**Comment by kvtim** [comment]
Links:
[index.html](https://kvtim.github.io/design/html/index)
[create.html](https://kvtim.github.io/design/html/create.html)
[posts.html](https://kvtim.github.io/design/html/posts.html)
[favorites.html](https://kvtim.github.io/design/html/favorite.html)

**Review by nastik1811 — COMMENTED** [review]
А у вас была заготовка или это вы так оперативно с нуля сделали?

**Comment by kvtim** [comment]
По урокам сначала делал и потом дорабатывал

File: design/html/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css"> 
+  <link rel="stylesheet" href="../css/base.css">
+  <link rel="stylesheet" href="../css/index.css">
+  <title>Blog</title>
+</head>
+<body>
+  <article>
+   <h1 class="title">Blog</h1>
+   <h2 class="subtitle">Create posts and add them to favorites</h2>
+   <button onclick="window.location.href='create.html'" class="button">Start</button>
```
Comment (nastik1811) [inline design/html/index.html]: It should be a link (`<a>`)

File: design/html/posts.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <li><a class="tab active" href="posts.html">Posts</a></li>
+      <li><a class="tab" href="create.html">Create</a></li>
+      <li><a class="tab" href="favorite.html">Favorites</a></li>
+    </ul>
+  </nav>
+
+  <div id="posts">
+    <div class="panel">
+      <div class="panel-head">
+        <h1>Post title</h1>
```
Comment (nastik1811) [inline design/html/posts.html]: h2

File: design/html/favorite.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <li class="tag tag-blue">News</li>
+        </ul>
+      </div>
+      <div class="panel-body">
+        <p>Content</p>
+      </div>
+      <div class="panel-footer">
+        <small>12.12.12</small>
+      </div>
+    </div>
```
Comment (nastik1811) [inline design/html/favorite.html]: We agreed this would be an article 👀

File: design/html/create.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      <li><a class="tab active" href="create.html">Create</a></li>
+      <li><a class="tab " href="favorite.html">Favorites</a></li>
+    </ul>
+  </nav>
+
+  <form id="create">
+    <h1>Create new post</h1>
+
+    <div class="form-control">
+      <label>Title</label>
```
Comment (nastik1811) [inline design/html/create.html]: labels should have `for` attributes

**Review by nastik1811 — CHANGES_REQUESTED** [review]
Some fixes are required  🐰

File: design/html/favorite.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <ul class="tabs">
+      <li><a class="tab " href="posts.html">Posts</a></li>
+      <li><a class="tab" href="create.html">Create</a></li>
+      <li><a class="tab active" href="favorite.html">Favorites</a></li>
+    </ul>
+  </nav>
+
+  <div id="favorite">
+    <article>
+      <div class="panel">
```
Comment (nastik1811) [inline design/html/favorite.html]: you don't need div here 
just `<article class="panel">`

Comment (kvtim) [reply design/html/favorite.html]: completed!

**Review by nastik1811 — APPROVED** [review]

**Pull request merged**